cypress/support/step-definitions/action-popover-steps.js

```diff
@@ -15,7 +15,7 @@ Then('Action Popover element is visible', () => {
 });
 
 Then('Action Popover element is not visible', () => {
-  actionPopover().should('not.be.visible');
```

Comment:
Should be `not.be.visible`, why did you change it ?

Reply:
It has to be `not.exist` 
https://files.slack.com/files-pri/T029WF4K9-F01J38HDHQW/image.png